functionnal_tests/cypress/integration/workspace/create_workspace_spec.js

```diff
@@ -25,19 +25,19 @@ describe('Workspace', () => {
       .should('exist')
   })
 
-  describe('Dashboard', () => {
-    describe('Creating two new workspaces', () => {
-      const createOneWorkspace = (cy, workspaceName) => {
-        return cy.get('[data-cy="sidebarCreateWorkspaceBtn"]').should('be.visible').click()
-          .get('[data-cy="createcontent__form__input"]').should('be.visible').type(workspaceName)
-          .get('[data-cy="popup__createcontent__form__button"]').should('be.visible').click()
-      }
+  const createOneWorkspace = (cy, workspaceName) => {
+    return cy.get('[data-cy="sidebarCreateWorkspaceBtn"]').should('be.visible').click()
+      .get('[data-cy="createcontent__form__input"]').should('be.visible').type(workspaceName)
+      .get('[data-cy="popup__createcontent__form__button"]').should('be.visible').click()
```

Comment:
if you are not inside a test, don't use should
Use should only if you are in an `it()`

Reply:
I remove the 3 `should` because it was useless to check if it was visible before do an action to the element selected, but here the const `createOneWorkspace` is only used in test case with it() so is it still invalid ?

Reply:
what do you mean by "invalid" ?
Is it that the function is only called once ? If yes, it's not a problem. If later we need this function, it'll be easier to refactor it at a higher level